data/persistence/src/desktopMain/kotlin/social/androiddev/common/persistence/di/persistenceModule.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * This file is part of Dodo.
+ *
+ * Dodo is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+ *
+ * Dodo is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with Dodo. If not, see <https://www.gnu.org/licenses/>.
+ */
+package social.androiddev.common.persistence.di
+
+import com.russhwolf.settings.PreferencesSettings
+import com.squareup.sqldelight.sqlite.driver.JdbcSqliteDriver
+import kotlinx.serialization.json.Json
+import org.koin.core.module.Module
+import org.koin.dsl.module
+import social.androiddev.common.persistence.AuthenticationDatabase
+import social.androiddev.common.persistence.localstorage.DodoAuthStorage
+import social.androiddev.common.persistence.localstorage.DodoAuthStorageImpl
+
+/**
+ * Koin DI module for all desktop specific persistence dependencies
+ */
+actual val persistenceModule: Module = module {
+    single<DodoAuthStorage> {
+        DodoAuthStorageImpl(
+            settings = PreferencesSettings
+                .Factory()
+                .create("DodoAuthSettings"),
+            json = Json {
+                ignoreUnknownKeys = true
+                encodeDefaults = false
+            }
+        )
+    }
+
+    single {
+        val driver = JdbcSqliteDriver(url = JdbcSqliteDriver.IN_MEMORY).also { driver ->
```

Comment:
I wonder why using in-memory db here? Data will be gone once app is closed, right?

Reply:
Hm not sure - that actually seems like a bug. @thebino was this an oversight when you created this sqlite driver for desktop?